labs/lab4/chapter4.md

```diff
@@ -30,14 +30,14 @@ docker port wordpress
 
 and taking look at the "PORTS" column for the wordpress site. You 
 can also get your ip address by looking at the address for the 
-eth1 interface after you execute:
+eth0 interface after you execute:
 
 ```bash
-ip addr show dev eth1
+minishift ssh "ip addr show dev eth0"
```

Comment:
honestly either address should work so we can just use `minishift ip` and not worry about explaining the difference